app/javascript/src/components/Dashboard/Contacts/Table.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from "react";
+
+import { Table } from "@bigbinary/neetoui/v2";
+
+import { RECORDS_LENGTH, CONTACTS, COLUMNS } from "./constants";
+
+const ContactsTable = () => {
```

Comment:
Please don't suffix with common component names. 
Also, wouldn't `ListContacts` be a better name here?